backend/actions/event.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from "express";
+import { Event } from "../models/Event";
+import mongoose from "mongoose";
+
+export const AddEvent = async (req: Request, res: Response) => {
+  const { email, time, eventName, description } = req.body;
+
+  try {
+    const result = await new Event({
+      email,
+      time,
+      eventName,
+      description,
+    }).save();
+    res.send({ res: "Success", data: result });
+  } catch (err) {
+    console.log(err);
```

Comment:
Since you're logging errors, it would be better choice to use `console.error` here.
It's not that different in normal environment but if you connect the application to the monitoring application, it's really different handling the logs and error messages.